test/box-luatest/net_box_test.lua

```diff
@@ -314,3 +315,24 @@ g.test_box_error = function()
         c.unprepare, c, "asd")
     c:close()
 end
+
+g.test_gc = function()
+    local c = net.connect(g.server.net_box_uri)
+    t.assert(c:eval('return true'))
+    local weak_refs = setmetatable({
+        conn = c,
+        callback = c._callback,
+        transport = c._transport,
+    }, {__mode = 'v'})
+    t.assert(weak_refs.conn)
+    t.assert(weak_refs.callback)
+    t.assert(weak_refs.transport)
+    c = nil -- luacheck: no unused
+    fiber.sleep(0)
+    collectgarbage('collect')
+    fiber.sleep(0)
```

Comment:
Nit: `fiber.sleep(0) == fiber.yield()` in Lua.

Reply:
Fixed.